packages/wallet-sdk/src/sign/scw/utils/findOwnerIndex.ts

```diff
@@ -1,10 +1,8 @@
 import { Client, Hex, isAddress, pad } from 'viem';
 import { readContract } from 'viem/actions';
-
-import { standardErrors } from ':core/error/errors.js';
 import { abi } from './constants.js';
 
-export async function getOwnerIndex({
+export async function findOwnerIndex({
```

Comment:
owner could be in the initcode? check this updated version that i made before https://github.com/coinbase/coinbase-wallet-sdk/blob/e835e14eff815c266f479ca4307b8cdb600cd15b/packages/wallet-sdk/src/sign/scw/utils/getOwnerIndex.ts